<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,7,2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_19" stretch="1,8">
           <item>
            <widget class="QGroupBox" name="groupBox_11">
             <property name="title">
              <string>显示控制</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="balise_tableWidget"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_dataRecv">
         <attribute name="title">
          <string>数据接收</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="6,5,4">
            <item>
             <widget class="QGroupBox" name="groupBox_12">
              <property name="title">
               <string>采样数据</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="MyCustomPlot" name="widget_sampleData" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_13">
              <property name="title">
               <string>平均频谱</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="MyCustomPlot" name="widget_fre" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_6"/>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_14">
              <property name="title">
               <string>误码率</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="MyCustomPlot" name="widget_errorBit" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_7"/>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_msgRecv">
         <attribute name="title">
          <string>原始报文</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QTextEdit" name="textEdit_msg">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_msgAnalyse">
         <attribute name="title">
          <string>报文解析结果</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11" stretch="3,1,1,7,4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>信号频率(MHZ)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>FL</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit">
               <property name="text">
                <string>3.951</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>FC</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="text">
                <string>4.234</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>FH</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="text">
                <string>4.516</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>实际</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_FL"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>实际</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_FC"/>
             </item>
             <item row="1" column="4">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>实际</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QLineEdit" name="lineEdit_FH"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>偏差</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_FL_bias"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>偏差</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="lineEdit_FC_bias"/>
             </item>
             <item row="2" column="4">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>偏差</string>
               </property>
              </widget>
             </item>
             <item row="2" column="5">
              <widget class="QLineEdit" name="lineEdit_FH_bias"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>数据平均传输率(Kb/s)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>标准</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_11">
               <property name="text">
                <string>564.48</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>实际</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_bps"/>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>偏差</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLineEdit" name="lineEdit_bps_bias"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>信号振幅和功率</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>振幅</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_amp"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>功率</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_power"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>应答器报文接收情况</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>总比特数</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_totalBitNum"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>总报文数</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="lineEdit_totalMsgNum"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>数据包数</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_17"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>逻辑编码</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="lineEdit_18"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>作用范围</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_20"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>平均误码</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="lineEdit_19"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>列车速度</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_22"/>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>列车里程</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QLineEdit" name="lineEdit_21"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_28">
                 <property name="text">
                  <string>正确比特起始位置</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_29">
                 <property name="text">
                  <string>连续正确比特总数</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_30">
                 <property name="text">
                  <string>连续正确报文长度</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLineEdit" name="lineEdit_firstCorrectBitPos">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_totalCorrectBitNum"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_correctMSGNum"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>修改采样门限和采样长度</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,1" columnminimumwidth="1,2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>采样门限</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>采样长度</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="pushButtonChangeOk">
               <property name="text">
                <string>确认修改</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_gate">
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>1/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>11/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>12/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>13/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>14/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>15/16 满摆幅</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>满摆幅</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_length">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <item>
                <property name="text">
                 <string>持续20bit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>持续50bit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>持续100bit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>持续200bit</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据回放</string>
    </property>
    <addaction name="action_bin"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>显示控制</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>采样数据</string>
     </property>
     <addaction name="action_show_original_sampleDataWave"/>
     <addaction name="action_show_sampleDataWave"/>
     <addaction name="action_show_phaseWave"/>
    </widget>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menua">
    <property name="title">
     <string>参数设置</string>
    </property>
    <addaction name="action_config_fre"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menua"/>
  </widget>
  <action name="action_bin">
   <property name="text">
    <string>导入.bin文件</string>
   </property>
  </action>
  <action name="action_show_original_sampleDataWave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示原始采样数据</string>
   </property>
  </action>
  <action name="action_config_fre">
   <property name="text">
    <string>频谱估计</string>
   </property>
  </action>
  <action name="action_show_sampleDataWave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示采样数据</string>
   </property>
  </action>
  <action name="action_show_phaseWave">
   <property name="text">
    <string>显示相位折线</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">mycustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
